idaes/generic_models/unit_models/column_models/column.py

```diff
@@ -0,0 +1,1144 @@
+
+        self.liquid_phase.apply_transformation()
+
+        # Add Ports for vapor side
+        self.add_inlet_port(name="vapor_inlet", block=self.vapor_phase)
+        self.add_outlet_port(name="vapor_outlet", block=self.vapor_phase)
+
+        # Add Ports for liquid side
+        self.add_inlet_port(name="liquid_inlet", block=self.liquid_phase)
+        self.add_outlet_port(name="liquid_outlet", block=self.liquid_phase)
+
+    # ==========================================================================
+        """ Add performace equation method"""
+        self._make_performance()
+
+    def _make_performance(self):
+        """
+        Constraints for unit model.
+
+        Args: None
+
+        Returns: None
+
+        """
+
+        # ======================================================================
+        # Custom Sets
+        vap_comp = self.config.vapor_side.property_package.component_list
+        liq_comp = self.config.liquid_side.property_package.component_list
+        equilibrium_comp = vap_comp & liq_comp
+        solvent_comp_list = \
+            self.config.liquid_side.property_package.solvent_set
+        solute_comp_list = self.config.liquid_side.property_package.solute_set
+        liq_comp_apparent = self.config.liquid_side.property_package._apparent_set
+        vapor_phase_list_ref = \
+            self.config.vapor_side.property_package.phase_list
+        liquid_phase_list_ref = \
+            self.config.liquid_side.property_package.phase_list
+
+        # Packing  parameters
+        self.eps_ref = Param(initialize=0.97,units=None,
+                             mutable=True,
+                             doc="Packing void space m3/m3")
+
+        self.packing_specific_area = Param(initialize=250,units=pyunits.m**2 / pyunits.m**3,
+                           mutable=True,
+                           doc="Packing specific surface area (m2/m3)")
+        
+        self.packing_channel_size = Param(initialize=0.1,units=pyunits.m,
+                           mutable=True,
+                           doc="Packing channel size (m)")
+        
+        self.hydraulic_diameter = Expression(expr=4 * self.eps_ref / self.packing_specific_area,
+                                 doc="Hydraulic diameter (m)")
+
+        # Add the integer indices along vapor phase length domain
+        self.zi = Param(self.vapor_phase.length_domain, mutable=True,
+                        doc='''Integer indexing parameter required for transfer
+                             across boundaries of a given volume element''')
+                             
+        # Set the integer indices along vapor phase length domain
+        for i, x in enumerate(self.vapor_phase.length_domain, 1):
+            self.zi[x] = i
+
+        # Unit Model Design Variables
+        # Geometry
+        self.diameter_column = Var(domain=Reals,
+                                   initialize=0.1,
+                                   units=pyunits.m,
```

Comment:
This should wait until we've cleaned up everything else, but the units of all these variables should be drawn from one of the property packages.

See lines 257 and 277 in https://github.com/IDAES/idaes-pse/blob/main/idaes/generic_models/unit_models/column_models/solvent_condenser.py for an example.